tests/phpunit/testdata/plugins/test-plugin-code-obfuscation-sourceguardian-errors/obfuscated.php

```diff
@@ -0,0 +1,4 @@
+<?php
+if(function_exists('sg_load')) {
```

Comment:
Even though this violates WPCS, this must not be changed since that is the string to look for to detect Source Guardian. See the original check linked in the PR description.

It's a test plugin anyway, which is excluded from linting for reasons like this.